src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -12,28 +13,27 @@ public boolean hasSomebodyWon() {
         return (hasPlayerWonSet(playerOne, playerTwo) || hasPlayerWonSet(playerTwo, playerOne));
     }
 
-    public boolean hasSomeoneOfPlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
-        return (playerWhoIsWinning.getSetScore() - playerWhoIsLoosing.getSetScore() == 2);
+    public boolean hasOneOfThePlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLosing) {
+        return (playerWhoIsWinning.getSetScore() - playerWhoIsLosing.getSetScore() == 2);
     }
 
     public boolean hasPlayerWonSet(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
```

Comment:
Also as the method above:

1. Since it doesn't affect the state of the instances of this class, you could have this method as a static method.
2. Since it's used as a helper method only within this class, you could have it private instead of public.